### PR TITLE
Announce Lagom 1.3.10

### PR DESCRIPTION
**Pull request by TimMoore**

### CODE REVIEW

File: src/blog/lagom-1-3-10.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+This release provides a few bug fixes, minor improvements and dependency updates. As an overview, this release contains the following:
+
+- Play updated to 2.5.18, fixing a security issue (see [Security Advisory](#security-advisory) below)
+- Akka Persistence Cassandra updated to 0.30, fixing an issue where [events can be skipped in a read-side processor](https://github.com/lagom/lagom/issues/1032)
+- You can now use [collection-value path parameters](https://github.com/lagom/lagom/issues/643) (for example, http://localhost:9000/users?username=alice&username=bob&username=carlos)
+- Fixes a [compatibility issue](https://github.com/lagom/lagom/issues/1018) with Firefox and streaming service calls
+
+
+We've continued to get great pull requests from the Lagom community. Thanks to GitHub users [Jirka Kremser](https://github.com/Jiri-Kremser), [Micho](https://github.com/micho10), [Jules Ivanic](https://github.com/guizmaii), [Olivier Nouguier](https://github.com/cheleb), [Saniya Tech](https://github.com/saniyatech), [Julien Jean Paul Sirocchi](https://github.com/sirocchj) and [HSalaila](https://github.com/HSalaila) for your contributions to Lagom 1.3.10!
```
Comment (rstento) [inline src/blog/lagom-1-3-10.md]: Maybe "Special thanks to ..." if there were others who contributed but aren't mentioned?

Comment (TimMoore) [reply src/blog/lagom-1-3-10.md]: This is everyone who sent a pull request that was included, but there are many ways to contribute, so I'll take your suggestion. Thanks! 😄

**Comment by TimMoore** [comment]
Lagom 1.3.10 is now available in Maven Central, so this can be merged when ready.

**Comment by TimMoore** [comment]
I'll go ahead and merge this so we can announce it today.

**Pull request merged**